testonly/hammer/hammer.go

```diff
@@ -85,6 +85,34 @@ const (
 
 var mapEntrypoints = []MapEntrypointName{GetLeavesName, SetLeavesName, GetSMRName, GetSMRRevName}
 
+// Constants for both valid and invalid operation choices.
+type Choice string
+const (
+	ExistingKey    = Choice("ExistingKey")
```

Comment:
consider making these plain errors with `errors.New`.  They are both strings and you wouldn't need to introduce a new type.

Reply:
I disagree: these aren't conceptually `error`s, they're indications of which operation to perform.  Also, stronger typing is better.